vocab/properties.json

```diff
@@ -631,7 +625,7 @@
         ]
     },
     "fullDocumentation": {
-        "description": null,
+        "description": "Usually printed instructions, comments, and information for using a particular product.",
```

Comment:
I suggest removing "Usually printed". Maybe replace with "Non-abridged"

Reply:
Yes. Good idea. I'll change it